CHANGELOG.md

```diff
@@ -1,5 +1,76 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.7.1
+
+<!-- generated comparing v5.7.0..master -->
+
+_May 17, 2022_
+
+A big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Blog post for announcing the release of the Premium plan of MUI X is out @joserodolfofreitas.
+- And more 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.7.1`
+
+- &#8203;<!-- 37 -->[Autocomplete] Fix `getInputProps` TypeScript return type (#32730) @ZeeshanTamboli
+- &#8203;<!-- 36 -->[Autocomplete] Forward props to renderTags() (#32637) @emlai
+- &#8203;<!-- 35 -->[Badge] Fix TypeScript error when adding style overrides for Badge (#32745) @ZeeshanTamboli
+- &#8203;<!-- 09 -->[Menu] Fix context menu open position (#32661) @oliviertassinari
+
+### `@mui/system@5.7.1`
+
+- &#8203;<!-- 05 -->[system] Add `Container` component and `createContainer` factory (#32263) @mnajdova
```

Comment:
Do we consider this as a new API, if yes the version should be `v5.8.0`? cc @mnajdova

Reply:
Addressed!